Library/Homebrew/test/test_cmd_audit.rb

```diff
@@ -301,10 +301,10 @@ class Foo < Formula
   end
 
   # Regression test for https://github.com/Homebrew/homebrew/pull/48744
-  # Formulae with "++" in their name would break the name check because of a
-  # regexp error:
+  # and https://github.com/Homebrew/homebrew/pull/48744
```

Comment:
No worries! That happens to all of us every once in a while. Good proof you're not a bot. :smile_cat: